pkg/builder/kaniko/utils_test.go

```diff
@@ -22,10 +22,10 @@ func TestCreateTarGz(t *testing.T) {
 	}
 	for file := range expectedFiles {
 		filePath := filepath.Join(testDir, file)
-		err := os.MkdirAll(filepath.Dir(filePath), os.ModePerm)
+		err := os.MkdirAll(filepath.Dir(filePath), 0600)
```

Comment:
why instead of predefined constant a hard coded value is used?

Reply:
Defined one locally since there are not constants defined in the go libraries.

Reply:
Thanks.
I see. that `os.ModePerm` is I think `0777` like full perms.

Reply:
correct, but that is the only one. They don't have all the combos.